network/endpoint.go

```diff
@@ -82,6 +85,7 @@ type RouteInfo struct {
 	Protocol int
 	DevName  string
 	Scope    int
+	Priority int
```

Comment:
can we keep this as metric and instead, and add a comment to say that it corresponds to route metric?

Reply:
golang netlink package names it as priority..keeping it consistent with that